### PR TITLE
Reduced the severity of the inconsistency message while adding/removing elements in Visual Tree on WASM.

### PR DESCRIPTION
**Pull request by carldebilly**
Fixes https://github.com/unoplatform/uno/issues/6988

### CODE REVIEW

**Review by jeromelaban — APPROVED** [review]

**Review by davidjohnoliver — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/UIElement.netstd.cs [hunk 1]
```diff
@@ -170,7 +170,7 @@ private void OnChildAdded(UIElement child)
 
 			if (child.IsLoaded)
 			{
-				this.Log().Error($"{this}: Inconsistent state: child {child} is already loaded (OnChildAdded)");
+				this.Log().Debug($"{this.GetDebugName()}: Inconsistent state: child {child} is already loaded (OnChildAdded). Common cause for this is an exception during Unloaded handling.");
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/UIElement.netstd.cs:173]: Enclose with `if(this.Log().IsDebug()` to avoid the unnecessary formatting.

**Comment by carldebilly** [comment]
ci: apply PR review
